lib/styles/settings/colors/_tokens-bodywork-variables.scss

```diff
@@ -0,0 +1,41 @@
+:root {
+	--primary-text: var(--theme-700);
```

Comment:
Don't we need similar file for WNL tokens? According to my knowledge, they're also theme specific

Reply:
All wnl variables are in default file `_tokens-variables.scss`. We don't have wnl tokens, we have default and bodywork tokens. I think it's not a problem now.